.github/workflows/canary-integration-test-arm64.yml

```diff
@@ -16,67 +16,67 @@ jobs:
       BLOCK: /dev/sdb
 
     steps:
-    - name: checkout
-      uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
+      - name: checkout
```

Comment:
Going to revert the whitespace changes in the actions? Or if we're changing the standard for whitespace, I think a separate PR for whitespace would be better.